src/test/java/org/apache/sling/jcr/repoinit/PrincipalBasedAclTest.java

```diff
@@ -852,7 +864,12 @@ public void testRemoveAllPrincipalMismatch() throws Exception {
         setup = "set principal ACL for otherSystemPrincipal\n"
                 + "remove * on " + path + "\n"
                 + "end";
-        U.parseAndExecute(setup);
+        try {
+            U.parseAndExecute(setup);
+            fail("Setting a principal ACL outside a supported path must not succeed");
+        } catch (RuntimeException e) {
+            // expected
```

Comment:
see above

Reply:
done in https://github.com/apache/sling-org-apache-sling-jcr-repoinit/pull/14/commits/cfadffb925ae3da29838a5f7a6a15630a147fbec